Libraries/LibWeb/HTML/HTMLElement.cpp

```diff
@@ -422,28 +422,26 @@ String HTMLElement::get_the_text_steps()
     // 6. Replace each remaining run of consecutive required line break count items
     //    with a string consisting of as many U+000A LF code points as the maximum of the values
     //    in the required line break count items.
+    StringBuilder builder;
     for (size_t i = 0; i < results.size(); ++i) {
-        if (!results[i].has<RequiredLineBreakCount>())
+        if (auto* string = results[i].get_pointer<String>()) {
```

Comment:
Could we use `results[i].visit()` rather than having `get` and `get_pointer` calls?

Reply:
totes